src/module_type_translator.ts

```diff
@@ -286,36 +326,72 @@ export class ModuleTypeTranslator {
     // goog.requireType types, which allows using them in type annotations
     // without causing a load.
     //   const requireTypePrefix = goog.requireType(moduleNamespace)
-    this.additionalImports.push(ts.factory.createVariableStatement(
-        undefined,
-        ts.factory.createVariableDeclarationList(
-            [ts.factory.createVariableDeclaration(
-                requireTypePrefix, /* exclamationToken */ undefined,
-                /* type */ undefined,
-                ts.factory.createCallExpression(
+    this.additionalImports.push(
+      es6modules
+        ? ts.factory.createImportDeclaration(
+            /* modifiers */ undefined,
+            /* importClause */ ts.factory.createImportClause(
+              /* isTypeOnly */ false,
+              /* name */ undefined,
+              /* namedBindings */ ts.factory.createNamespaceImport(
+                ts.factory.createIdentifier(requireTypePrefix)
+              )
+            ),
+            /* moduleSpecifier */ ts.factory.createStringLiteral(
+              moduleNamespace
+            )
+          )
+        : ts.factory.createVariableStatement(
```

Comment:
When es6modules is true, use an import declaration.  If not, use goog.requireType